index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Sharp" />
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap"
+        rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+</head>
+
+<body>
+    <nav>
+        <div class="top-bar">
+            <ul>
+                <a href="#">
+                    <li><i class="uil uil-facebook-f"></i></li>
+                </a>
+                <a href="#">
+                    <li><i class="uil uil-twitter"></i></li>
+                </a>
+                <a href="#">
+                    <li>English</li>
+                </a>
+                <a href="#">
+                    <li>My Page</li>
+                </a>
+                <a href="#">
+                    <li>Logout</i></li>
+                </a>
+            </ul>
+        </div>
+    </nav>
+    <header class="header">   
+        <div class="menu-links">
+            <ul>
+                <a href="#" class="menu-link"><img src="./images/logo.png"></a> 
+
+                <a class="menu-link" href="./about.html">
+                    <li>About</li>
+                </a>
+                <a class="menu-link" href="#program">
+                    <li>Program</li>
+                </a>
+                <a class="menu-link" href="#join">
+                    <li>Join</li>
+                </a>
+                <a class="menu-link" href="#sponsor">
+                    <li>Sponsor</li>
+                </a>
+                <a class="menu-link" href="#news">
+                    <li>News</li>
+                </a>
+                <a class="menu-link" href="#speakers">
+                    <li>WHO Campaign</li>
+                </a>
+
+            </ul>
+        </div>
+        <button class="humburger">
+            <div class="bar"></div>
+        </button>

```

Comment:
- [ ] As mentioned by the previous reviewer:
 **_kindly ensure that your navbar maintains its shape and position throughout all screen sizes of the desktop_**
🛑  your navbar still changes on different screen sizes 🛑 


![Bugcapstone](https://user-images.githubusercontent.com/89282221/210779102-c93e0314-aaaa-44a8-b920-0e6a979a7ea9.gif)